guides/_subsections/create-a-custom-target.md

```diff
@@ -0,0 +1,84 @@
+## [Create a Custom Target](#create-a-custom-target)
+
+By default, Cryostat automatically discovers target JVMs over remote Java Management Extensions (JMX), using various mechanisms (e.g. Kubernetes API, JDP). However, in some cases (e.g. for Kubernetes API, JMX port is not `9091` and port name is not `jfr-jmx`), Cryostat might not see your applications. Fortunately, you can tell Cryostat about them by specifying Custom Targets by filling out the Custom Target form.
+
+<ol>
+  <li>
+    {% include_relative _subsections/common/navigate-to-topology.md %}
+  </li>
+  <li>
+    {% capture open-custom-target-form-text %}
+    <p>
+      Search for Custom Target tab and click <i>Create</i> to open the Custom Target form.
```

Comment:
We should consider italicizing or bolding *Custom Target* since it is a special data form. From the current state of the guides, we should move forward with some consistent formatting, maybe it seems data types should be bolded, and navigation/button text should be italicized, and maybe user-named data (e.g. recording Names) should be enclosed in code blocks? 

Currently everything special is italicized and these guides are the first to introduce bolding for data.

Reply:
Good idea I will file an issue!

Reply:
https://github.com/cryostatio/cryostatio.github.io/issues/120